src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.container {
+  height: 100vh;
+  display: flex;
```

Comment:
I think it's better to sort styles like [that](https://codeguide.academy/html-css.html#css-order) 
Because it's a bit confusing to find padding at the end.

Reply:
🔥